<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cardfinder</class>
 <widget class="QDialog" name="Cardfinder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget_cards">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>319</width>
     <height>215</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>319</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Your bonus cards</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать карту</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить карту</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_history">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>История операций</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pay">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Оплатить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
